### DISCUSSION TITLE
Comparing Files in Visual Studio Code

### DISCUSSION THREAD

**Discussion by mmueller2012** [Sysmagine/SemanticDiff#26] [Blog]
Share your opinions and questions on our blog post [Comparing Files in Visual Studio Code](https://semanticdiff.com/blog/visual-studio-code-compare-files/).

**Comment by HoomanT** [comment]
Thank you!

**Reply by slackner** [reply]
Glad you found the article helpful!

**Comment by JanBergsma** [comment]
Thanks!
